kubernetes/controller/Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN go mod download
 COPY cmd/main.go cmd/main.go
 COPY api/ api/
 COPY internal/ internal/
+COPY plugins/ plugins/
```

Comment:
none of this needs to be exposed out of internal

Reply:
no. this is completely unrelated. if you have a module level internal package (that already exists) you can consume them from main.